core/templates/pages/story-editor-page/story-preview-tab/story-preview-tab.component.html

```diff
@@ -1,23 +1,23 @@
 <div class="parent-container">
   <div class="oppia-story-viewer-title">
-    <[$ctrl.story.getTitle()]>
+    {{ story.getTitle() }}
   </div>
   <div class="oppia-story-viewer-description">
-    <[$ctrl.story.getDescription()]>
+    {{ story.getDescription() }}
   </div>
   <div class="oppia-story-header-text">
     <strong>
-      1 out of <[$ctrl.nodes.length]> Chapters Completed
+      1 out of {{ nodes?.length }} Chapters Completed
     </strong>
   </div>
   <div class="d-md-block oppia-story-viewer-card-container">
-    <div ng-repeat="node in $ctrl.storyContents.getNodes() track by $index;"
+    <div *ngFor="let node of storyContents?._nodes let i = index;"
```

Comment:
same doubt as above.  Also @srijanreddy98  is it fine to change the name from `index` to `i` ?

Reply:
ditto

Reply:
It's fine.